app/api/groups/accept_or_reject_requests.go

```diff
@@ -0,0 +1,54 @@
+package groups
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/go-chi/render"
+
+	"github.com/France-ioi/AlgoreaBackend/app/database"
+	"github.com/France-ioi/AlgoreaBackend/app/service"
+)
+
+func (srv *Service) acceptOrRejectRequests(w http.ResponseWriter, r *http.Request) service.APIError {
+	parentGroupID, err := service.ResolveURLQueryPathInt64Field(r, "parent_group_id")
+	if err != nil {
+		return service.ErrInvalidRequest(err)
+	}
+
+	action, err := service.ResolveURLQueryGetStringField(r, "action")
+	if err != nil {
+		return service.ErrInvalidRequest(err)
+	}
+	if action != "accept" && action != "reject" {
+		return service.ErrInvalidRequest(errors.New("'action' should be either 'accept' or 'reject'"))
+	}
+
+	groupIDs, err := service.ResolveURLQueryGetInt64SliceField(r, "group_ids")
+	if err != nil {
+		return service.ErrInvalidRequest(err)
+	}
+
+	user := srv.GetUser(r)
+	if apiErr := checkThatUserOwnsTheGroup(srv.Store, user, parentGroupID); apiErr != service.NoError {
+		return apiErr
+	}
+
+	var results *database.GroupGroupTransitionResults
+	if len(groupIDs) > 0 {
+		err = srv.Store.InTransaction(func(store *database.DataStore) error {
+			results, err = store.GroupGroups().AcceptOrRejectRequests(parentGroupID, groupIDs, action == "accept")
+			return err
+		})
+	}
+
+	service.MustNotBeError(err)
+
+	response := service.Response{
+		Success: true,
+		Message: "updated",
```

Comment:
A common problem (like a DB error or something) will result in "Internal server error"

Reply:
Still, the remark was: I am sure it is ok to succeed if all operations inside have failed. For instance, if you accept 1 invitation and it fails (because it was withdrawn just before by another user), it will result in success. Is it really what we want?